changelogs/changelog.yaml

```
@@ -62,3 +70,5 @@ releases:
     fragments:
       - flightctl_1.0.0_update.yml
     release_date: '2025-05-15'
+  1.1.0:
+    release_date: '2025-06-15'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)